js/alert_banner.js

```diff
@@ -44,6 +44,33 @@
 
   Drupal.behaviors.AlertBannersRestBlock = {
     attach: function (context, settings) {
+      // Replace jQuery.cookie with modern alternative
```

Comment:
Consider using Drupal's cookie service for better integration:
```javascript
// Instead of native JS cookie handling
const setCookie = (name, value) => {
  document.cookie = `${name}=${value};path=/;SameSite=Lax;Secure`;
};
```
The current implementation works but adding SameSite and Secure attributes would improve security.